R/run-model.R

```diff
@@ -564,22 +564,22 @@ format_options <- function(options) {
   if (is.null(options$spectrum_artnum_calibration_level)) {
     options$spectrum_artnum_calibration_level <- "none"
   }
-  if (is.null(options$spectrum_artnum_strat)) {
+  if (is.null(options$spectrum_artnum_calibration_strat)) {
```

Comment:
These fixes should be tested. (Lack of testing of these functions is why the error happened in the first place...).

Let's get this fix merged, so approving without the tests. But can you create an issue to add tests for this function?

Reply:
Issue added here: #224